settings.gradle

```
@@ -1,4 +1,34 @@
 
+pluginManagement {
+    repositories {
+        maven {
+            url "${artifactory_contextUrl}/plugins-release"
+        }
+        if (gradlePluginsVersion.contains("SNAPSHOT") || versioningPluginVersion.contains("SNAPSHOT")) {
+            maven {
+                url "${artifactory_contextUrl}/plugins-snapshot-local"
+            }
+        }
+    }
+    plugins {
+        id 'org.labkey.build.antlr' version "${gradlePluginsVersion}" apply false
+        id 'org.labkey.build.api' version "${gradlePluginsVersion}" apply false
+        id 'org.labkey.build.base' version "${gradlePluginsVersion}" apply false
+        id 'org.labkey.build.distribution' version "${gradlePluginsVersion}" apply false
+        id 'org.labkey.build.fileModule' version "${gradlePluginsVersion}" apply false
+        id 'org.labkey.build.javaModule' version "${gradlePluginsVersion}" apply false
+        id 'org.labkey.build.jsdoc' version "${gradlePluginsVersion}" apply false
+        id 'org.labkey.build.module' version "${gradlePluginsVersion}" apply false
+        id 'org.labkey.build.multiGit' version "${gradlePluginsVersion}" apply false
+        id 'org.labkey.build.npmRun' version "${gradlePluginsVersion}" apply false
+        id 'org.labkey.build.serverDeploy' version "${gradlePluginsVersion}" apply false
+        id 'org.labkey.build.teamCity' version "${gradlePluginsVersion}" apply false
+        id 'org.labkey.build.testRunner' version "${gradlePluginsVersion}" apply false
+        id 'org.labkey.build.tomcat' version "${gradlePluginsVersion}" apply false
+        id 'org.labkey.build.xsddoc' version "${gradlePluginsVersion}" apply false
+    }
```

Do we need to specify the versioning plugin version?
```suggestion
        id 'org.labkey.versioning' version "${versioningPluginVersion}" apply false
    }
```

---

Yes, I wasn't sure that was going to work since we publish that plugin differently, but it seems to do the right thing.  I can apply this plugin to an individual module using the plugins closure in its build.gradle file. I'm not sure if we'll ever have need for that, but nice to have common treatment.